tests/gaslessTransaction.test.ts

```diff
@@ -0,0 +1,22 @@
+import {GasLessTransaction,VerifiedWallet, Provider} from '../src'
+// Only for testing
+const apiKey ='a6645c57b516720cc218e98a645887d59fa53b3e1e08fed0c58de17b3589b799'
+const rpcUrl = "https://api.stackup.sh/v1/node/a6645c57b516720cc218e98a645887d59fa53b3e1e08fed0c58de17b3589b799"
+const paymasterUrl = 'https://api.stackup.sh/v1/paymaster/a6645c57b516720cc218e98a645887d59fa53b3e1e08fed0c58de17b3589b799'
+
+describe("userop gasless transaction", () => {
+
+    test('gasless transaction should succeed', async() => { 
+        const mnemonics = await VerifiedWallet.generateMnemonic()
+        const wallet = VerifiedWallet.importWallet(mnemonics)
+        const walletWithProvider = wallet.setProvider(
+          Provider.stackUpProvider(rpcUrl)
+      )
+        const transaction = new GasLessTransaction({ apiKey,paymasterUrl,wallet, walletWithProvider});
```

Comment:
@herberthk 

I could not understand why you are initializing GasLessTransaction directly ? It should be used by the contracts in /src/contract. Can you check if any contract in /src/contract can be called using GasLessTransaction ?

Reply:
This was for testing purposes unfortunately, there is no complete usage sample of any of the contracts in the SDK, however basing on https://github.com/verified-network/verified-sdk#interacting-with-smart-contracts, I thought that's how it supposed to be

Reply:
@herberthk 

here is a snippet you can use to test, add the mnemonics in a .env file

import {} from "dotenv/config";
import {
  VerifiedWallet,
  Provider,
  SecuritiesFactory,
} from "@verified-network/verified-sdk";
import { ethers } from "ethers";

//REACT_APP_MNEMONICS = wallet mnemonics saved in env
const sender = VerifiedWallet.importWallet(process.env.REACT_APP_MNEMONICS);
//API_KEY = provider api e.g infura
//georli can be any network e.g "polygon"
const signer = sender.setProvider(
  Provider.infuraProvider("georli", process.env.API_KEY)
);
const factoryAddress = "0x647a7EA364a31fa18c65f4043CD55718d245D774"; //Securities Factory
//construct contract using it's required constructor in sdk
//factoryAddress = address of contract to initialize
const secFactoryContract = new SecuritiesFactory(signer, factoryAddress);

const zeroAddress = "0x0000000000000000000000000000000000000000";
const currency = "0x07865c6E87B9F70255377e024ace6630C1Eaa37F"; //usdc
const defaultBytes =
  "0x53544f434b000000000000000000000000000000000000000000000000000000"; //STOCK as bytes
const indiaBytes =
  "0x496e646961000000000000000000000000000000000000000000000000000000"; //India as bytes
const abiCoder = ethers.utils.defaultAbiCoder;
 const restrictions = [];
const encodedArray = abiCoder.encode(["bytes32[]"], [restrictions]);

await secFactoryContract
  .issueSecurity(
    zeroAddress,
    defaultBytes,
    web3.utils.asciiToHex("VERIFIED"),
    web3.utils.asciiToHex("tata"),
    currency,
    sender.address,
    sender.address,
    encodedArray,
    indiaBytes,
    false
  )
  .then((res) => {
    console.log("Security Issued Succesfully with hash: ", res.response.hash);
  })
  .catch(() => {
    console.error("issueSecurity failed");
  });

Reply:
@herberthk 

And yes, you are right. The current sdk uses callContract and what you can do is based override it with the gasless transaction function that will use userops. Basically, we want sdk users to choose whether they want to use ethers or userops

Reply:
@kallolborah You're right there is a need to implement `callContract` method in `verifiedContract` version for userop and I can do it, just consider this example
```ts
new ethers.Contract(ERC_TOKEN, ERC20_ABI, provider)
``` 
According to stackup documentation `ERC_TOKEN` is required while calling contract but for this use case `ERC_TOKEN` and `ERC20_ABI` are not required. 
So should I create it for future contracts that require `ERC_TOKEN` and `ERC20_ABI`?. What do you think?

Reply:
@herberthk 

I think you should just create a different version of callContract in /userops/index.js which uses exported urls for paymaster, bundler etc. callContract will be used by multiple sdk contracts that extend the VerifiedContract. We should not change the constructor of the VerifiedContract itself as it will break the way all sdk contracts are initialized.